processor/src/main/java/org/mapstruct/ap/internal/model/LifecycleCallbackMethodReference.java

```diff
@@ -109,12 +91,8 @@ public String getTargetVariableName() {
         return declaringType != null ? Collections.asSet( declaringType ) : java.util.Collections.<Type> emptySet();
     }
 
-    public List<Parameter> getParameterAssignments() {
-        return parameterAssignments;
-    }
-
     public boolean hasMappingTargetParameter() {
-        for ( Parameter param : parameterAssignments ) {
+        for ( Parameter param : getParameterBindings() ) {
```

Comment:
They represent what's bound to parameters of the method reference. Perhaps I should introduce a new class for that, e.g. `ParameterBinding` to differentiate it more clearly...